tests/integrations/database/db.py

```diff
@@ -75,3 +76,13 @@ def truncate_tables():
         else:
             logger.error(e)
             connection.rollback()
+
+
+def get_trace_data_with_retry(user_id: str, retry: int = 3) -> Optional[Dict[str, Any]]:
+    trace = get_trace_data(user_id)
+    count = 0
+    while count < retry and not trace:
+        count += 1
+        time.sleep(2)
+        trace = get_trace_data(user_id)
+    return trace
```

Comment:
the span_name is needed here https://github.com/GreptimeTeam/greptimeai/blob/main/tests/integrations/langchain_/test_with_openai_chat_models.py#L35-L40